WebDevStudios/nextjs-wordpress-starter
# Feature/26 design system

Closes https://github.com/WebDevStudios/nextjs-wordpress-starter/issues/26 (partially)

### Link

Coming soon...

### Description

* Add config values to the tailwind config file for:
  * Colors
  * Fonts
  * Font sizes
  * Spacing
  * Borders (width and radius)
* Add Button component
* Refactor Hero component, styles and story to match designs
* Refactor Alert component, styles and story to match designs
* Configure storybook

### Screenshot

Coming soon...

### Verification

How will a stakeholder test this?

1. Pull down this branch
2. Run `yarn storybook`
3. Read the documentation stories for colors, typography, spacing, Alert, Hero, Button
4. Make sure it all makes sense and matches the designs
